mock-mhs-adaptor/src/main/java/uk/nhs/adaptors/mockmhsservice/controller/MhsMockController.java

```diff
@@ -41,7 +41,8 @@
         try {
             String interactionId = Optional.ofNullable(headers.get("interaction-id")).orElse("");
             String correlationId = Optional.ofNullable(headers.get("correlation-id")).orElse("");
-            return mockMhsService.handleRequest(interactionId, correlationId, mockMhsMessage);
+            String waitForResponse = Optional.ofNullable(headers.get("wait-for-response")).orElse("");
```

Comment:
Could we use a constant "StringUtils.EMPTY" instead?

Reply:
fixed